release-notes/opensearch-custom-codecs.release-notes-3.2.0.0.md

```diff
@@ -0,0 +1,13 @@
+## 2025-08-19 Version 3.2.0.0
+
+Compatible with OpenSearch and OpenSearch Dashboards version 3.2.0
+
+### Features
+* Adding support for composite index ([#263](https://github.com/opensearch-project/custom-codecs/pull/263))
+
+### Infrastructure
+* Report code coverage to codecov ([#267](https://github.com/opensearch-project/custom-codecs/pull/267))
+* Bump gradle to 8.14, fix backport-deletion and support JDK24 ([#266](https://github.com/opensearch-project/custom-codecs/pull/266))
+
+### Maintenance
+* [AUTO] Increment version to 3.2.0-SNAPSHOT ([#262](https://github.com/opensearch-project/custom-codecs/pull/262))
```

Comment:
@gaiksaya @rishabh6788 Honest question, are including entries like this in the release notes useful? From what I've seen many repos have a policy of including every commit in the release notes with no exceptions. In the OpenSearch repo we allow wide discretion for excluding commits from the CHANGELOG (which means they won't go in the release notes), so commits like this are generally excluded. I have a hard time imagining how it is helpful to any consumer of the release notes to see a link to the commit that created the release notes document that I'm reading...

Reply:
@andrross apologies, merged right when you commented, please feel free to correct, thank you

Reply:
Agree @andrross !
JDK24 support makes sense but version increment does not provide any value I believe. 
We can exclude this kind of commits.

Reply:
@reta No worries! This was more of a general comment about how we could possibly improve the automation.